SConstruct

```diff
@@ -518,7 +518,7 @@ if selected_platform in platform_list:
     # are actually handled to change compile options, etc.
     detect.configure(env)
 
-    print(f'Building for platform "{selected_platform}", architecture "{env["arch"]}", target "{env["target"]}.')
+    print(f'Building for platform "{selected_platform}", architecture "{env["arch"]}", target "{env["target"]}".')
```

Comment:
Note that this is already fixed in master, so the change should just disappear in the merge commit.

Reply:
Might be worth rebasing then, it's weird to have two commits do the same change. Git seems not to complain but it's always confusing IMO.

Reply:
I rebased so it's cleaner and doesn't have conflicts when cherry-picking.